src/tests/predicates.test.js

```diff
@@ -489,6 +489,54 @@ describe('predicates.js', () => {
       assert.equal(1, result.failed.size);
     });
 
+    it('should not filter energidirect user with not_contains when value not set', () => {
```

Comment:
This is a public repo. We should scrub mentions like this.
```suggestion
    it('should not filter energidirect user with not_contains when value not set', () => {
```